File: app/src/main/java/com/zawzaw/savethelibrary/model/gson/GsonNew.java
package com.zawzaw.savethelibrary.model.gson;

import com.google.gson.annotations.SerializedName;

/**
 * Created by zawzaw on 25/12/17.
 */

public class GsonNew
{

 @SerializedName("post_id")
 private int post_id;

 @SerializedName("post_title")
 private String post_title;

 @SerializedName("post_content")
 private String post_content;

 @SerializedName("author_name")
 private String author_name;

 @SerializedName("date")
 private String date;

 @SerializedName("image")
 private String feature_image_path;

    public int getPost_id()
    {
        return post_id;
    }

    public String getPost_title()
    {
        return post_title;
    }

    public String getPost_content()
    {
        return post_content;
    }

    public String getAuthor_name()
    {
        return author_name;
    }

    public String getDate()
    {
        return date;
    }

    public String getFeature_image_path()
    {
        return feature_image_path;
    }

}

File: app/src/main/java/com/zawzaw/savethelibrary/model/gson/GsonNew.kt
package com.zawzaw.savethelibrary.model.gson

import com.google.gson.annotations.SerializedName

/**
 * Created by zawzaw on 31/12/17.
 */

class GsonNew {

    @SerializedName("post_id")
    val post_id: Int = 0

    @SerializedName("post_title")
    val post_title: String? = null

    @SerializedName("post_content")
    val post_content: String? = null

    @SerializedName("author_name")
    val author_name: String? = null

    @SerializedName("date")
    val date: String? = null

    @SerializedName("image")
    val feature_image_path: String? = null

}
